packages/react-components/priority-overflow/src/overflowManager.ts

```diff
@@ -162,6 +163,9 @@ export function createOverflowManager(): OverflowManager {
 
   const observe: OverflowManager['observe'] = (observedContainer, userOptions) => {
     Object.assign(options, userOptions);
+    observing = true;
```

Comment:
Should be this set to `false` in `disconnect`?

Reply:
yeah good catch